### PR TITLE
Update Argentina holidays. Add holidays localization: uk

### PR DESCRIPTION
**Pull request by KJhellico**

### CODE REVIEW

File: holidays/locale/uk/LC_MESSAGES/AR.po [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+msgstr "День пам'яті генерала Мартіна Мігеля де Гуемеса"
+
+#: holidays/countries/argentina.py:144
+msgid "Paso a la Inmortalidad del General Don Manuel Belgrano"
+msgstr "День пам’яті генерала Мануеля Бельграно"
+
+#. Independence Day
+#: holidays/countries/argentina.py:149
+msgid "Día de la Independencia"
+msgstr "День незалежності"
```
Comment (arkid15r) [inline holidays/locale/uk/LC_MESSAGES/AR.po:96]: Here it's lowercased. Just saying :)

**Review by arkid15r — COMMENTED** [review]
LGTM 🇦🇷 

Thanks for fixing the if/else mess!

**Comment by arkid15r** [comment]
@KJhellico could you also update README?

**Comment by KJhellico** [comment]
> Thanks for fixing the if/else mess!

In fact, at first I was going to do just that and nothing more…

**Comment by KJhellico** [comment]
> @KJhellico could you also update README?

Done. I forgot about it, thank you!

**Pull request merged**